src/components/NewMessageForm/NewMessageFormAttachments.vue

```diff
@@ -0,0 +1,179 @@
+<!--
+  - @copyright Copyright (c) 2022, Marco Ambrosini <marcoambrosini@icloud.com>
+  -
+  - @author Marco Ambrosini <marcoambrosini@icloud.com>
+  - @author Maksim Sukharev <antreesy.web@gmail.com>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<NcActions ref="attachmentsMenu"
+		:container="container"
+		:boundaries-element="boundariesElement"
+		:disabled="disabled"
+		:aria-label="t('spreed', 'Share files to the conversation')"
+		:aria-haspopup="true">
+		<template #icon>
+			<Paperclip :size="16" />
+		</template>
+
+		<NcActionButton v-if="canUploadFiles"
+			close-after-click
+			@click.prevent="$emit('open-file-upload')">
```

Comment:
The "open-file-upload" event has been triggered but not declared on `emits` option